### PR TITLE
User calibration Fixes

### PR DESCRIPTION
**Pull request by amrsoll**
Multiple fixes, see commit messages.

This is how the camera settings will look for 97% of our users when they do the update.

Screenshot from 2020-10-08 11-37-56

### CODE REVIEW

**Comment by amrsoll** [comment]
TODO @Iratxe: Fix the lens calibration exit when clicking on a button of the navbar.

**Comment by amrsoll** [comment]
New fix for the camera accuracy and just the settings in general

TODO : Set the camera to accurate during the corner calibration. Undo it after.

File: octoprint_mrbeam/static/js/settings/camera_settings.js [hunk 1]
```diff
@@ -120,56 +120,83 @@ $(function () {
...
+                    cam: {
+                        remember_markers_across_sessions: val === "reliable",
+                    },
+                };
+                $("#camera_settings_marker_detection button").prop(
+                    "disabled",
+                    true
+                );
+                OctoPrint.settings
+                    .savePluginSettings("mrbeam", _data)
```
Comment (amrsoll) [inline octoprint_mrbeam/static/js/settings/camera_settings.js:148]: This is the way to go to sync settings with the backend

**Comment by amrsoll** [comment]
@irlaec feel free to open again and merge when the calibration exits as expected :)

**Review by irlaec — APPROVED** [review]

**Comment by andywerner** [comment]
When you both agree on it, please go ahead an merge it.

**Review by andywerner — APPROVED** [review]

**Pull request merged**